tests/test_rwc_jazz.py

```diff
@@ -96,10 +121,20 @@ def test_load_metadata():
         'track_number': 'Tr. 04',
         'title': 'Crescent Serenade (Piano Solo)',
         'artist': 'Makoto Nakamura',
-        'duration_sec': '02:47',
+        'duration': 167,
         'variation': 'Instrumentation 1',
         'instruments': 'Pf',
     }
+    assert metadata['RM-J044'] == {
+        'piece_number': 'No. 44',
+        'suffix': 'M04',
+        'track_number': 'Tr. 09',
+        'title': 'Joyful, Joyful, We Adore Thee',
+        'artist': 'K’s Band',
```

Comment:
I think this is what's breakign the Circle CI tests - change the apostrophe in K's.